### DISCUSSION TITLE
Request system administrator privileges

### DISCUSSION THREAD

**Discussion by uiuing** [wailsapp/wails#1453] [Q&A]
### Is your feature request related to a problem? Please describe.

Does wails support an api for granting administrator rights to the entire application?

When creating a folder using os.Mkdir("/usr/local/test", 0777)
I encountered a problem with insufficient permissions

### Describe the solution you'd like

Hopefully I can request permissions for the whole process via a hook in wails

**Comment by leaanthony** [comment]
Wails does not provide an API for this. Try this: https://github.com/lu4p/go-escalate
